casbin/enforcer.py

```diff
@@ -240,3 +279,7 @@ def delete_roles_for_user_in_domain(self, user, role, domain):
     def get_permissions_for_user_in_domain(self, user, domain):
         """gets permissions for a user or role inside domain."""
         return self.get_filtered_policy(0, user, domain)
+
+    def get_permissions_for_user_by_named_policy_in_domain(self, ptype, user, domain):
+        """gets permissions for a user or role with named policy inside domain."""
+        return self.get_filtered_named_policy(ptype, 0, user, domain)
```

Comment:
Please reuse code in `get_implicit_permissions_for_user` and `get_permissions_for_user_in_domain`.

Reply:
ok